idd/Energy+.idd.in

```diff
@@ -56118,10 +56118,35 @@ Coil:Cooling:WaterToAirHeatPump:EquationFit,
         \units W
         \autosizable
    N5,  \field Gross Rated Cooling COP
+        \note Gross cooling COP at rated conditions
         \required-field
         \type real
         \units W/W
         \minimum> 0.0
+   N6,  \field Rated Entering Water Temperature
+        \note Rated entering water temperature corresponding to the water-to
+        \note -air application for which this coil is used. For example: for water loop
+        \note applications, the rated temperature is 30 degree Celsius
+        \units C
+        \type real
+        \minimum> 0
+        \default 30
+   N7,  \field Rated Entering Air Dry-Bulb Temperature
+        \note Rated entering air dry-bulb temperature corresponding to the
+        \note water-to-air application for which this coil is used. For example: for
+        \note water loop applications, the rated temperature is 27 degree Celsius
+        \units C
+        \type real
+        \default 27
+        \minimum> 0
+   N8,  \field Rated Entering Air Wet-Bulb Temperature
+        \note Rated entering air wet-bulb temperature corresponding to the
+        \note water-to-air application for which this coil is used. For example: for
+        \note water loop applications, the rated temperature is 19 degree Celsius
+        \units C
+        \type real
+        \default 19.0
+        \minimum> 0
```

Comment:
I guess it was agreed to put these here, but in general, if you have new fields that can be defaulted, going at the end of the object would avoid having to write transition rules.  And hopefully soon we'll be fully EpJSON, so the order you put them in right now is not as critical as it used to be.

Reply:
That makes sense, I'll keep that in mind for next time -- if we're not fully EpJSON by then!